internal/version/version.go

```diff
@@ -14,16 +14,17 @@
 
 package version
 
-import (
-	"fmt"
-	"runtime"
-)
-
-//var needs to be used instead of const for ldflags
+// var needs to be used instead of const for ldflags
 var (
 	Version           = "unknown"
 	GitVersion        = "unknown"
 	GitCommit         = "unknown"
 	KubernetesVersion = "unknown"
-	GoVersion         = fmt.Sprintf("%s %s/%s", runtime.Version(), runtime.GOOS, runtime.GOARCH)
```

Comment:
`GoVersion` contains redundant information (GOOS and GOARCH are already printed by the `version` subcommand) so I removed it in favor of calling `runtime.Version()` directly.